lib/orderbook/OrderBook.ts

```diff
@@ -28,6 +28,8 @@ class OrderBook extends EventEmitter {
   private ownOrders: { [ pairId: string ]: Orders } = {};
   private peerOrders: { [ pairId: string ]: Orders } = {};
 
+  private localIds: { [ localId: string ]: string } = {};
```

Comment:
I'd rather call this `localIdMap` and maybe add a comment block explaining that this maps local orders Ids to global order Ids.

Also, I think this is virtually identical functionally speaking but I think it's a little more appropriate to define this as a `Map<string, string>`. That's the pattern we use for `responseMap` in the Peer class.